pkg/volume/glusterfs/glusterfs.go

```diff
@@ -840,11 +721,10 @@ func (p *glusterfsVolumeProvisioner) Provision() (*v1.PersistentVolume, error) {
 	gidStr := strconv.FormatInt(int64(gid), 10)
 
 	pv.Annotations = map[string]string{
-		volumehelper.VolumeGidAnnotationKey:        gidStr,
-		volumehelper.VolumeDynamicallyCreatedByKey: heketiAnn,
-		glusterTypeAnn:                             "file",
-		"Description":                              glusterDescAnn,
-		v1.MountOptionAnnotation:                   "auto_unmount",
```

Comment:
yeah, looks like this dropped https://github.com/rootfs/kubernetes/commit/04bf95a5d1b2251eba5df159a6b222fd99333c7a#diff-e97253dd603331ffca81131a4b67264fR869 accidentally